pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -1,29 +1,25 @@
 # -*- coding: utf-8 -*-
 from pokemongo_bot.event_manager import EventHandler
 from pokemongo_bot.base_dir import _base_dir
-import json
-import os
 import time
 import telegram
 import thread
 import re
-from pokemongo_bot.datastore import Datastore
 import pprint
+from pokemongo_bot.datastore import Datastore
```

Comment:
I think it is needed for database operations, as already explained on the other PR where the same question was asked. You are welcome to remove it and test!